src/components/pages/Step4.vue

```diff
@@ -180,6 +180,33 @@
                     v-bind:items="openSimModels"
                   />
               </v-card-text>
+
+              <v-card-title class="justify-center data-title">
+                <span class="mr-2">Select marker augmenter model</span>
+                <v-tooltip bottom="" max-width="500px">
+                  <template v-slot:activator="{ on }">
+                    <v-icon v-on="on"> mdi-help-circle-outline </v-icon>
+                  </template>
+                  OpenCap uses an LSTM model, also called marker augmenter model, to predict the 3D position of 43 anatomical markers from the 3D position of 20 video keypoints (https://www.biorxiv.org/content/10.1101/2022.07.07.499061v1). 
+                  The anatomical markers are used as input to OpenSim to compute joint angles using inverse kinematics.
+                  <br><br>
+                  The default model (v0.2) underwent training using 708 hours of motion capture data, yielding an RMSE of 4.8 +/- 0.2 deg (OpenPose and HRNet) for joint angles across 18 degrees of freedom. 
+                  <br><br>
+                  The latest model (v0.3) was trained on 1475 hours of motion capture data and resulted in an RMSE of 4.4 +/- 0.3 deg (OpenPose) and 4.1 +/- 0.3 deg (HRnet) for joint angles across 18 degrees of freedom.
```

Comment:
The latest model (v0.3) is more accurate and more robust to different activities than v0.2. We recommend using it for new studies -- it will become the default model in the future. It was trained on 1475 hours of motion capture data and resulted in an RMSE of 4.4 +/- 0.3 deg (OpenPose) and 4.1 +/- 0.3 deg (HRnet) for joint angles across 18 degrees of freedom.